sources/data-node/api/v2/trading_data.proto

```diff
@@ -107,6 +111,39 @@ message PageInfo {
     string end_cursor = 4;
 }
 
+// -- Positions --
+// Request for a list of positions for a party
+// Optionally, if a market identifier is set, the results will be filtered for that market only
+message GetPositionsByPartyRequest {
+    // Party identifier, required field
+    string party_id = 1;
+    // Market identifier
+    string market_id = 2;
+}
+
+// Response for a list of positions for a party
+message GetPositionsByPartyResponse {
+    // A list of 0 or more positions
+    repeated vega.Position positions = 1;
+}
```

Comment:
As above, I don't think you need these as they're already available in the v1 definition and we're deprecating this so we don't need to move it to the v2 definitions.

Reply:
will do